tests/test_suite.py

```diff
@@ -30,6 +30,11 @@
 
 from command import Command
 
+is_ubuntu_22_04 = False
+if sys.platform == 'linux':
+    result = subprocess.run(['lsb_release', '-sr'], stdout=subprocess.PIPE)
+    is_ubuntu_22_04 = result.stdout.decode().strip() == '22.04'
+
```

Comment:
This is just a minor style issue: there is a double initialization here, instead we should set `is_ubuntu_22_04` to `False` in a `else` statement.